internal/blockchain/ethereum/ethereum.go

```diff
@@ -431,13 +436,32 @@ func parseContractLocation(location fftypes.Byteable) (*Location, error) {
 	return ethLocation, nil
 }
 
+func parseParamDetails(details fftypes.Byteable) (*paramDetails, error) {
+	ethParam := &paramDetails{}
+	if err := json.Unmarshal(details, &ethParam); err != nil {
+		return nil, fmt.Errorf("failed to validate param")
```

Comment:
Should these use error codes in the translations file? This will be something that every blockchain plugin will need to do

Reply:
Added some localizations